internal/stack/compose.go

```diff
@@ -59,19 +46,16 @@ func dockerComposePull(options Options) error {
 		Services: withIsReadyServices(withDependentServices(options.Services)),
 	}
 
+	fmt.Printf("Docker-compose opts: %#v\n", opts)
```

Comment:
Was this for debugging or did you intend to leave this in? If it's the latter, maybe we should use `logger.Debugf` instead?